subworkflows/local/align.nf

```diff
@@ -50,26 +50,31 @@ workflow ALIGN {
 
     } else if (params.aligner == 'bwa') {
 
-        BWA_ALN (
+        // BWA_ALN (
```

Comment:
You can remove this rather than just comment it out. It'll always be in the git history if you need it again.

Reply:
yes -- done